blog/2020/03/01/ClassDiagram.svg

```diff
@@ -0,0 +1,341 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Can we rename this png and svg file to "flightSeatAssignmentClassDiagram.png" & svg to be consistent with the other filenames in other blog posts? Having it descriptive helps to keep it up to date (in case we want to reuse it in docs, presentations, etc)